src/main/resources/jenkins/branch/ItemColumn/column.jelly

```diff
@@ -25,26 +25,28 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <td style="${indenter.getCss(job)}">
+  <d:taglib uri="local">
+    <d:tag name="link">
+      <a href="${jobBaseUrl}${job.shortUrl}" class='model-link jenkins-table__link'>
+        <l:breakable value="${h.getRelativeDisplayNameFrom(job, itemGroup)}"/>
+      </a>
+    </d:tag>
+  </d:taglib>
+
+  <td style="${indenter.getCss(job)}" xmlns:local="local">
     <j:choose>
       <j:when test="${it.isOrphaned(job)}">
         <s>
-          <a href="${jobBaseUrl}${job.shortUrl}" class='model-link inside jenkins-table_link' title="${it.getTitle(job)}">
```

Comment:
@janfaracik Why did you remove the `inside` class?

Reply:
It doesn't do anything from what I've seen from core.

Reply:
It controlled whether the context menu dropdown chevron was considered part of the area of the `model-link`. If it doesn't do anything anymore, that's an oversight in https://github.com/jenkinsci/jenkins/pull/6084.

2.332.x `inside`: 
<img width="113" alt="Screenshot 2024-10-29 at 13 23 29" src="https://github.com/user-attachments/assets/f4117aa9-96f7-4448-b292-7065c56f3e50">

2.332.x otherwise (ugly but this hover effect was fairly short-lived):
<img width="83" alt="Screenshot 2024-10-29 at 13 23 22" src="https://github.com/user-attachments/assets/46ed8e4c-d8ea-4392-9635-d533893aa2fc">